tests/issues/118/sap-hana-preconfigure-issue-118-assert.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: all
+  tasks:
+  - name: Check if required packages are installed
+    shell: yum info installed {{ __sap_hana_preconfigure_packages | map('quote') | join(' ') }} | awk '/Name/{n=$NF}/Version/{v=$NF}/Release/{r=$NF}/Description/{printf ("%s\n", n)}'
```

Comment:
Can you use `package_facts` https://docs.ansible.com/ansible/2.9/modules/package_facts_module.html#package-facts-module or `rpm` instead of `yum`?

Reply:
I think I once gave up on using package_facts because I failed to handle the output appropriately.
In another piece of code as part of #104 , I used:
shell: rpm -q --qf "%{NAME}\n" $(echo {{ __sap_hana_preconfigure_packages }} | sed "s.,..g;s,[,,;s,],,;s,',,g") | grep -v "is not installed"
instead, which worked perfect. As this code is only used as part of a test, are you OK with leaving it in place for now?